components/board/PostCard.tsx

```
@@ -1,33 +1,306 @@
+          <Button
+            variant="ghost"
+            size="sm"
+            className={`flex items-center ${
+              hasVoted(post.id) ? "text-blue-600" : "text-gray-500"
+            } ${viewOnly ? "cursor-default" : ""} col-start-5 px-2`}
+            onClick={handleVote}
+          >
+            <ThumbsUp className="h-4 w-4 mr-2" />
+            <span>{post.voteCount}</span>
+          </Button>
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)